Geometry/HGCalGeometry/interface/HGCalGeometry.h

```diff
@@ -71,6 +71,7 @@ class HGCalGeometry final : public CaloSubdetectorGeometry {
                   CaloSubdetectorGeometry::IVec& dinsVector) const override;
 
   GlobalPoint getPosition(const DetId& id) const;
```

Comment:
having only one `getPosition` method with the `bool debug = false` as default would have done the same without duplication